src/Matterhorn/Types.hs

```diff
@@ -1917,6 +1917,9 @@ data MHError =
     -- ^ The specified script was not found
     | NoSuchHelpTopic T.Text
     -- ^ The specified help topic was not found
+    | BadFile T.Text
```

Comment:
A suggestion on this: rename to `BadAttachmentPath` to be more specific.

Reply:
(and maybe consider including the exception, or more detail about the nature of the badness)

Reply:
(and then the code to construct the error message text can just build the right text for the error)

Reply:
Thanks, made those improvements!